Resources/config/services.xml

```diff
@@ -21,5 +21,8 @@
             <argument></argument> <!-- active locales -->
             <argument></argument> <!-- active domains -->
         </service>
+        <service id="Bazinga\Bundle\JsTranslationBundle\Command\DumpCommand">
```

Comment:
omitting the class name to rely on the id is not possible, as the bundle supports Symfony DI older than 3.3.

Reply:
Thanks, I just fixed it